### DISCUSSION TITLE
Invalid bound column name

### DISCUSSION THREAD

**Discussion by DCPSTech** [eclipse-birt/birt#1085] [Q&A]
![multiple_exceptions_occured](https://user-images.githubusercontent.com/115656797/195407252-55eb3ca0-3e3a-4e59-b971-b5c6c726b224.png)

We used BIRT to design a Shipping Label for our BMC Remedy system. However when users go to print the label, sometimes they'll receive the above error. We aren't sure what to make of it and have tried looking over the report design to no success. It happens intermediately, so it's hard to pinpoint what's causing it and how to recreate it. I'm not sure how to post the code for the design, but any advice would be appreciated.

**Comment by hvbtup** [comment]
A typical reason for this kind of error is when you remove or rename a column in a data set and you forget to correct the binding in the layout items which are bound to that data set.

**Reply by DCPSTech** [reply]
We thought that as well and tried re-adding the layout items and reimporting the design into Remedy, but still receive the error.

**Comment by hvbtup** [comment]
Then you should upload your rptdesign including the *.rptlibrary and *.js files it references in a ZIP here.
